tests/integration/image/test_utils.py

```diff
@@ -157,122 +157,118 @@ def test_edge_to_image_size_vert_horz():
     assert actual == desired
 
 
-def test_extract_batch_size():
-    batch_size = 3
+class TestExtract:
```

Comment:
No need to group here. Although they all start with `extract`, they are not closely related. Please revert this.